### ISSUE TITLE
found a problem when using this repo

### ISSUE THREAD

**Issue by DamonsJ** [hoanganhken97/react-native-lyric#6]
if one line lyrics is too long, it can not show all the lyrics

**Comment by lovegaoshi** [comment]
yeah its bc each lrcLine's Text doesnt have width set to 100%; and its View has a hardcoded height = lineHeight. once you loose them both you can show multiple lines
